server_installation/topics/operator/external-database.adoc

```diff
@@ -0,0 +1,61 @@
+
+=== External database support
+
+Enabling external Postgresql database support requires creating `keycloak-db-secret` in the following form (note that values are Base64 encoded, see https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets[Kubernetes Secrets manual]):
+
+.`keycloak-db-secret` Secret
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+    name: keycloak-db-secret
+    namespace: keycloak
+stringData:
+    POSTGRES_DATABASE: <Database Name>
+    POSTGRES_EXTERNAL_ADDRESS: <External Database IP or URL (resolvable by K8s)>
+    POSTGRES_EXTERNAL_PORT: <External Database Port>
+    # Strongly recommended to use <'Keycloak CR Name'-postgresql>
+    POSTGRES_HOST: <Database Service Name>
+    POSTGRES_PASSWORD: <Database Password>
+    # Required for AWS Backup functionality
+    POSTGRES_SUPERUSER: true
+    POSTGRES_USERNAME: <Database Username>
+ type: Opaque
+```
+
+There are two properties specifically responsible for specifying external database hostname or IP address and its port:
+
+* POSTGRES_EXTERNAL_ADDRESS - an IP address (or a Hostname) or the external database. This address needs to be resolvable from Kubernetes cluster perspective.
```

Comment:
1) Typo - ```s/or the external database/of the external database/```
2) Rephrase `This address needs to be resolvable from Kubernetes cluster perspective` to `This address needs to be resolvable from the perspective of a Kubernetes cluster.`

Reply:
Fixed.